react-first-app/src/app/components/App/index.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
```

Comment:
This file is unnecessary, Remove it and use the Game component directly in `index.js`